packages/core/npm-aqua-compiler/src/imports.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Arborist from "@npmcli/arborist";
+import { breadth } from "treeverse";
+
+export async function gatherImportsFromNpm(
+  path: string,
+): Promise<Record<string, string[]>> {
```

Comment:
This does not return imports according to the latest spec.
https://www.notion.so/fluencenetwork/Aqua-Imports-Spec-082d48d2d7184b05a3f47f35b781c843
Type of new imports is `Record<string, Record<string, string[] | string>>`. Read more in the doc.

Reply:
Yeah I totally forgot we decided on a different structure. Thanks for the catch.